client/src/shared/codeEditor/codeTokenizer/tokenizeLine/parseSymbol.ts

```diff
@@ -0,0 +1,74 @@
+import createTokenizedLineWithError from './createTokenizedLineWithError';
+
+const IS_SYMBOL_REGEX = /^[A-Za-z]\w*$/;
+const IS_FUNCTION_REGEX = /^([A-Za-z]\w*)\((.*)\)$/;
+
+const parseSymbol = ({ symbol, values, customFunctions, lineString, functions }) => {
```

Comment:
I think it would be worth adding some tests for invalid function parameters and for redefining function name. I guess these test could be added to `tokenizeCodeRaw.test.ts`, but a separate test file for `parseSymbol` function would probably also make sense.

Reply:
Done